deliverables/README.md

```diff
@@ -8,61 +8,154 @@ To answer questions, please just put the answer on the next line after the quest
+  bottom: 5px;
+  height: 100px;
+  width:100px;
+  }
+
+**What are the three different ways to include CSS in your project or use CSS to style a particular element? **     
+CSS can be put in the HTML file via in-line:
+  Ex: <h1 style="color:red">Big Header Title</h1>
+This only changes the element it is written in line with.
+Internal:
+  <style>
+      selector{
+        property:value;
+      selector{
+        property:value;
+        }
+  </style>
+This will style all of the elements defined by the selectors in the HTML file.
+
+External:
+In a seperate CSS file.
+selector{
+  attribute:calue;
+  }
+This is ideal, as it keeps the html file clean and is more efficient.
+
+**Android Logo**  
+I did this
 
 ---
 ### Section 3: Git and Github  
-What is Git?  
-What is SCM?  
-What is a VCS?  
-Why is Git useful for a developer?  
-Why is Git useful for a team of developers?  
-How do you create a new Git repository for a project locally?  
-How do you create it on Github?  
-How do you commit changes?  
-What is the difference between staging and committing changes?  
-What is the difference between committing your changes and pushing them to Github?  
-What is the command to check the status of your current repo in git?  
-What is the command to see the history of your previous commits (from the command line)?  What are a few interesting variations (sets of options)?  
-How can you look through your historical commits on the Github website?  
-What is a "Merge"?  
-What is a "Pull Request"?  
-What is "Forking" a repo?  
-What is "Cloning" a repo?  
+**What is Git?**  
+Git is a platform that allows developers to collaborate and track their progress. It is especially helpful for its ability to store files as snapshots over time and work locally.
+
+**What is SCM?**  
+Software Configuration Management
+
+**What is a VCS?**  
+A VCS allows developers to see each version of their file over time.  This allows them to explore different options and recover older versions without any consequences.
+
+**Why is Git useful for a developer?**  
+Collaboration, snapshot style saving, collaboration, ability to work locally.
+
+**Why is Git useful for a team of developers?**  
+Git allows developers to collaborate on a project together, but still experiment and work individually.  Different members can see each other's progress and have projects delegated betwen them.
+
+**How do you create a new Git repository for a project locally?**  
+git init - This creates a new repo in the current directory.  With <directory> it creates it in a new folder with just this new repo.
+
+**How do you create it on Github?**  
++ in top right corner.
+
+**How do you commit changes?**  
+Click commit.
+
+**What is the difference between staging and committing changes?**  
+Staging allows you to edit your commits and format them before committing them.  You can then commmit changes to different edits individually without having to commit them before editing another area.
```

Comment:
an analogy:  staging is putting certain things in the box, committing is taping the box shut and putting a label on it